tasks/snyk_findings/snyk_findings.rb

```diff
@@ -154,14 +143,6 @@ def run(opts)
               "tags" => [project.fetch("source"), project.fetch("packageManager")]
 
             }
```

Comment:
these blocks are just removed? I just added them to fix some issues with the locator info

Reply:
Where in the code did you call tags again?  Rubocop says they are not used.

Reply:
tags is added to the asset hash. i'll check the code again.